src/machines/modelingMachine.ts

```diff
@@ -1493,6 +1450,63 @@ export const modelingMachine = setup({
         return {} as SetSelections
       }
     ),
+    extrudeAstMod: fromPromise<
+      unknown,
+      ModelingCommandSchema['Extrude'] | undefined
+    >(async ({ input }) => {
+      if (!input) return new Error('No input provided')
```

Comment:
Should probably use `Promise.reject` ? Or is an error not a failed promise? Trying it in the NodeJS REPL it comes back as a resolved promise with an `Error` value

Reply:
Yeah I think you're right I think I goofed